Backpack/Color/Classes/Generated/BPKColor.m

```diff
@@ -292,6 +292,15 @@ + (UIColor *)clear {
   return UIColor.clearColor;
 }
 
++ (UIColor *) label {
+if (@available(iOS 13.0, *)) {
```

Comment:
This code is gonna be used a lot. Probably worth having a helper method that takes the two colours and spits back the right one.